packages/vm/lib/state/interface.ts

```diff
@@ -22,7 +22,7 @@ export interface StateManager {
   checkpoint(): Promise<void>
   commit(): Promise<void>
   revert(): Promise<void>
-  getStateRoot(): Promise<Buffer>
+  getStateRoot(force: boolean): Promise<Buffer>
```

Comment:
Correct me if I am wrong, but I think `force` should be marked as `optional` in the interface, since there is a default value in the implementation.

Reply:
I can try this, I tried using a default value in the interface (`false`) but TypeScript does not support this. Marking it optional in the interface sounds like a good idea, I think that might work, but I don't know if TypeScript supports these optional values in the interface. Will try.